src/routes/solid-router/advanced-concepts/lazy-loading.mdx

```diff
@@ -15,8 +15,8 @@ const Home = () => import("./Home");
 const Users = lazy(() => import("./Users"));

```

Comment:
should Home be wrapped in lazy too?

Reply:
It should, yes.

@snnsnn do you mind updating the example to include `lazy`?

Reply:
Sorry for the delay. I’ve fixed it and added a few empty lines to make it more readable.